common/v2/features/ReceiveAssets/ReceiveAssets.tsx

```diff
@@ -116,6 +116,11 @@ const ErrorMessage = styled.span`
 
 export function ReceiveAssets({ history }: RouteComponentProps<{}>) {
   const { accounts } = useContext(AccountContext);
+  const shouldRedirect = accounts === undefined || accounts === null || accounts.length === 0;
+  if (shouldRedirect) {
+    return <Redirect to="/no-accounts" />;
```

Comment:
Prefer path from https://github.com/MyCryptoHQ/MyCrypto/blob/gau/common/v2/config/routePaths.ts#L125. For example ROUTE_PATHS.NO_ACCOUNTS.path instead of hardcoded one.